broker-daemon/proto/broker.proto

```diff
@@ -664,8 +665,30 @@ message GetTradeHistoryResponse {
  * ```
  */
 service OrderService {
+  /** CreateOrder creates a new order that gets passed directly to the Relayer,
+   * which returns an order ID and then the order gets saved to the local
+   * database with the order ID as key.
+   *
+   * ```javascript
+   * const { orderId } = await orderService.createOrder({
+   *   sourceAmount: {
+   *     symbol: 'BTC',
+   *     value: '0.01'
+   *   },
+   *   sourceAddress: 'sb1qsakedujs4exdq5dz2cpldcph6hlvf82l8mn8uk',
```

Comment:
these addresses of the wrong format. They should be payment channel network addresses of the kind returned by `LndEngine#getPaymentChannelNetworkAddress`